front/app/components/SeatInfo/index.tsx

```diff
@@ -42,6 +42,14 @@ const SeatInfo = ({ seatType }: Props) => {
   };
   const maximumSeatNumber = maximumSeatNumbers[seatType];
 
+  const additionalAdmins =
+    appConfiguration?.data.attributes.settings.core.additional_admins_number;
+  const additionalCollaborators =
+    appConfiguration?.data.attributes.settings.core
+      .additional_moderators_number;
+  const maximumAdditionalSeats =
+    seatType === 'admin' ? additionalAdmins : additionalCollaborators;
```

Comment:
Not blocking, but maybe we can also use the object version we're using for the other values in SeatInfo?